Netskope/netskope_events/ingest/parser.yml

```diff
@@ -36,7 +36,7 @@ stages:
           "@timestamp": "{{parse_date.datetime}}"
           observer.vendor: "Netskope"
           event.dataset: "{{parsed_event.message.type}}"
-          event.action: "{{parsed_event.message.activity}}"
+          event.action: "{{parsed_event.message.action or parsed_event.message.activity or 'Allow'}}"
```

Comment:
Sadly, we have to keep `parsed_event.message.activity` into `event.action`.
Can you move `parsed_event.message.action` into `action.name`? Thank you